### PR TITLE
Reformat and fix warnings

### PR DESCRIPTION
**Pull request by TroubleMakerBen**
* fixes all but one warning
* brings formatting closer to what we have in other classes

### CODE REVIEW

File: MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m [hunk 1]
```diff
@@ -224,10 +220,12 @@ + (MSAppleErrorLog *)errorLogFromCrashReport:(MSPLCrashReport *)report {
   // All errors are fatal for now, until we add support for handled exceptions.
   errorLog.fatal = YES;
 
-  // appLaunchTOffset - the difference between crashtime and initialization time, so the "age" of the crashreport before
-  // it's forwarded to the channel.
-  // We don't care about a negative difference (will happen if the user's time on the device changes to a time before
-  // the crashTime and the time the error is processed).
+  /**
```
Comment (clpolet) [inline MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m]: /*

File: MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m [hunk 2]
```diff
@@ -240,19 +238,23 @@ + (MSAppleErrorLog *)errorLogFromCrashReport:(MSPLCrashReport *)report {
     }
   }
 
-  // errorLog.architecture is an optional. The Android SDK will set it while for iOS, the file will be set on the
-  // server using primaryArchitectureId and architectureVariantId.
+  /**
```
Comment (clpolet) [inline MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m]: /*

File: MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m [hunk 3]
```diff
@@ -240,19 +238,23 @@ + (MSAppleErrorLog *)errorLogFromCrashReport:(MSPLCrashReport *)report {
...
+   * errorLog.architecture is an optional. The Android SDK will set it while for iOS, the file will be set on the
+   * server using primaryArchitectureId and architectureVariantId.
+   */
 
-  // HockeyApp didn't use report.exceptionInfo for this field but exception.name in case of an unhandled exception or
-  // the report.signalInfo.name. More so, for BITCrashDetails, we used the exceptionInfo.exceptionName for a field called exceptionName.
+  /**
```
Comment (clpolet) [inline MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m]: This comment's formatting is a bit weird :)

File: MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m [hunk 4]
```diff
@@ -106,7 +106,7 @@ static NSInteger bit_binaryImageSort(id binary1, id binary2, void *__unused cont
   for (unsigned int i = 0; i < images_count; ++i) {
```
Comment (clpolet) [inline MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m]: In this file there are many comments  in code blocks using
```
/**
 * Multiline 
 * comment
 */
```

Where it should instead be
```
/*
 * Multiline 
 * comment
 */
```

File: MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m [hunk 5]
```diff
@@ -409,21 +421,25 @@ + (NSNumber *)calculateTOffsetFromReport:(MSPLCrashReport *)report {
     lastException.frames = exceptionThread.frames;
     lastException.type = report.exceptionInfo.exceptionName ?: report.signalInfo.name;
 
-    // Don't add the thread to the array of threads (as in HockeyApp), the exception will be added to the crashed thread instead.
+    /** Don't add the thread to the array of threads (as in HockeyApp), the exception will be added to the crashed
```
Comment (clpolet) [inline MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m]: Review this comment formatting.

File: MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m (hunk 4 above)
Comment (jaelim-ms) [inline MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m]: Yes. `/**` is only for documentation.

Comment (TroubleMakerBen) [inline MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m]: Oh. Okay.

**Pull request merged**